src/code/code_80097A00.c

```diff
@@ -78,136 +78,141 @@ u32 gGsFlagsMasks[] = { 0x000000FF, 0x0000FF00, 0x00FF0000, 0xFF000000 };
+    gItemIconDekuNutTex,             // ITEM_DEKU_NUT
+    gItemIconBombTex,                // ITEM_BOMB
+    gItemIconBowTex,                 // ITEM_BOW
+    gItemIconArrowFireTex,           // ITEM_ARROW_FIRE
+    gItemIconDinsFireTex,            // ITEM_DINS_FIRE
+    gItemIconSlingshotTex,           // ITEM_SLINGSHOT
+    gItemIconOcarinaFairyTex,        // ITEM_OCARINA_FAIRY
+    gItemIconOcarinaOfTimeTex,       // ITEM_OCARINA_OF_TIME
+    gItemIconBombchuTex,             // ITEM_BOMBCHU
+    gItemIconHookshotTex,            // ITEM_HOOKSHOT
+    gItemIconLongshotTex,            // ITEM_LONGSHOT
+    gItemIconArrowIceTex,            // ITEM_ARROW_ICE
+    gItemIconFaroresWindTex,         // ITEM_FARORES_WIND
+    gItemIconBoomerangTex,           // ITEM_BOOMERANG
+    gItemIconLensOfTruthTex,         // ITEM_LENS_OF_TRUTH
+    gItemIconMagicBeansTex,          // ITEM_MAGIC_BEANS
+    gItemIconHammerTex,              // ITEM_HAMMER
+    gItemIconArrowLightTex,          // ITEM_ARROW_LIGHT
+    gItemIconNayrusLoveTex,          // ITEM_NAYRUS_LOVE
+    gItemIconBottleEmptyTex,         // ITEM_BOTTLE_EMPTY
+    gItemIconBottlePotionRedTex,     // ITEM_BOTTLE_POTION_RED
+    gItemIconBottlePotionGreenTex,   // ITEM_BOTTLE_POTION_GREEN
+    gItemIconBottlePotionBlueTex,    // ITEM_BOTTLE_POTION_BLUE
+    gItemIconBottleFairyTex,         // ITEM_BOTTLE_FAIRY
+    gItemIconBottleFishTex,          // ITEM_BOTTLE_FISH
+    gItemIconBottleMilkFullTex,      // ITEM_BOTTLE_MILK_FULL
+    gItemIconBottleRutosLetterTex,   // ITEM_BOTTLE_RUTOS_LETTER
+    gItemIconBottleBlueFireTex,      // ITEM_BOTTLE_BLUE_FIRE
+    gItemIconBottleBugTex,           // ITEM_BOTTLE_BUG
+    gItemIconBigBottlePoeTex,        // ITEM_BOTTLE_BIG_POE
+    gItemIconBottleMilkHalfTex,      // ITEM_BOTTLE_MILK_HALF
+    gItemIconBottlePoeTex,           // ITEM_BOTTLE_POE
+    gItemIconWeirdEggTex,            // ITEM_WEIRD_EGG
+    gItemIconCuccoTex,               // ITEM_CUCCO
+    gItemIconZeldasLetterTex,        // ITEM_ZELDAS_LETTER
+    gItemIconMaskKeatonTex,          // ITEM_MASK_KEATON
+    gItemIconMaskSkullTex,           // ITEM_MASK_SKULL
+    gItemIconMaskSpookyTex,          // ITEM_MASK_SPOOKY
+    gItemIconMaskBunnyTex,           // ITEM_MASK_BUNNY
+    gItemIconMaskGoronTex,           // ITEM_MASK_GORON
+    gItemIconMaskZoraTex,            // ITEM_MASK_ZORA
+    gItemIconMaskGerudoTex,          // ITEM_MASK_GERUDO
+    gItemIconMaskTruthTex,           // ITEM_MASK_TRUTH
+    gItemIconSoldOutTex,             // ITEM_SOLD_OUT
+    gItemIconPocketEggTex,           // ITEM_POCKET_EGG
+    gItemIconPocketCuccoTex,         // ITEM_POCKET_CUCCO
+    gItemIconCojiroTex,              // ITEM_COJIRO
+    gItemIconOddMushroomTex,         // ITEM_ODD_MUSHROOM
+    gItemIconOddPotionTex,           // ITEM_ODD_POTION
+    gItemIconPoachersSawTex,         // ITEM_POACHERS_SAW
+    gItemIconBrokenBiggoronSwordTex, // ITEM_BROKEN_BGS
```

Comment:
This one's inconsistent too (and IMO neither name is correct, see comment elsewhere).